src/language-css/printer-postcss.js

```diff
@@ -155,7 +155,12 @@ function genericPrint(path, options, print) {
 
       return [
         node.raws.before.replace(/[\s;]/g, ""),
-        insideICSSRuleNode(path) ? node.prop : maybeToLowerCase(node.prop),
+        (parentNode.type === "css-atrule" &&
+          typeof parentNode.raws.params === "string" &&
+          parentNode.raws.params.startsWith(":")) ||
```

Comment:
Should we use `parentNode.variable` instead?

Reply:
Ah, fixed cases with comments  https://github.com/prettier/prettier/pull/14034/commits/fd414fbd8dd79bc809f465c19374aec170e2b1c0

Reply:
`parentNode.variable` seems fine/better